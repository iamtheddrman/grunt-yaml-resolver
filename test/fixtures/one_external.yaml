local: This is local
external:
  $ref: ./simple.yaml