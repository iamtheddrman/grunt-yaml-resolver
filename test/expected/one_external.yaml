local: This is local
external:
  info: {subitem: This is a test}
  otherInfo: This is also a test
